src/Backoffice/Task/App/Requests/UpsertTaskRequest.php

```diff
@@ -42,7 +42,9 @@ public function toDto(): TaskDto
         return new TaskDto(
             title: $this->string(self::TITLE)->toString(),
             description: $this->string(self::DESCRIPTION)->toString(),
-            status: $this->string(self::STATUS)->toString(),
+            status: $this->enum(self::STATUS, TaskStatus::class) ?? TaskStatus::from(
+                $this->string(self::STATUS)->toString()
+            ),
```

Comment:
Since the status field is already validated with Rule::enum(TaskStatus::class), the fallback with ?? TaskStatus::from(...) is unnecessary here.
You can safely simplify this to:
`status: $this->enum(self::STATUS, TaskStatus::class)`

Reply:
I think she did this because of phpstan, I didn't realized this.

Reply:
If it's a PHPStan issue, I recommend define it as a variable, type it and then use it in the DTO

```suggestion
            /** @var TaskStatus $status */
            $status = $this->enum(self::STATUS, TaskStatus::class);

            return new TaskDto(
                title: $this->string(self::TITLE)->toString(),
                description: $this->string(self::DESCRIPTION)->toString(),
                status: $status
            );
```